src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/position.html.twig

```diff
@@ -24,5 +24,9 @@
  *#}
 
 <div class="text-center">
-  {{ record[column.options.position_field] + 1 }}
+  {% if column.options.increment_position %}
+    {{ record[column.options.position_field] + 1}}
```

Comment:
Now I'm doubting this original display, since the position definition can define their first position I don't see why we change it here, it's probably my mistake from when I first developed this feature

Wether the position are 0-indexed or 1-indexed I guess we should always display it without modifications, if one needs to have them indexed on 1 then he just needs to configure the definition properly I guess